pkg/plugin/conftest/plugin_test.go

```diff
@@ -734,6 +669,148 @@ func TestPlugin_ParseConfigAuditReportData(t *testing.T) {
 	}))
 }
 
+func TestPlugin_ParseConfigAuditReportDataWithTitleAndIDOnMetadata(t *testing.T) {
```

Comment:
In Go, I'd typically group test cases as subtests or use table driven test cases:

```go
func ParseConfigAuditReportData(t *testing.T) {

	t.Run("WithTitleOnMetadataOnly", func(t *testing.T) {
		// ...
	});

	t.Run("WithTitleAndIDOnMetadata", func(t *testing.T) {
		// ...
	});
}
```